app/src/ui/history/commit-list-item.tsx

```diff
@@ -353,6 +357,11 @@ export class CommitListItem extends React.PureComponent<
         label: 'Copy SHA',
         action: this.onCopySHA,
       },
+      {
+        label: 'Copy Tags',
```

Comment:
I think this label should change depending on the platform (both words capitalized on macOS, but not on Windows) and the number of tags (singular vs plural). Probably worth extracting that logic to a variable prior to this chunk of code 😅